src/khoj/interface/web/config.html

```diff
@@ -194,6 +194,36 @@ <h3 class="card-title">API Keys</h3>
                 </button>
             </div>
         </div>
+        <div id="phone-number-input-id" class="api-settings">
+            <div class="card-title-row">
+                <img class="card-icon" src="/static/assets/icons/whatsapp.svg" alt="WhatsApp icon">
+                <h3 class="card-title">WhatsApp</h3>
+            </div>
+            <div class="card-description-row">
+                {% if phone_number %}
+                    <p id="api-settings-card-description" class="card-description">Your number is connected. You can now send messages to Khoj on WhatsApp.</p>
```

Comment:
Does the `phone_number` being set mean that verification is complete too? The web_client doesn't seem to gate passing the `phone_number` behind a  `verified_phone_number` check.

Maybe good to pass `verified_phone_number` state from web_client to config.html. And update this conditional?

```suggestion
                {% if phone_number and verified_phone_number %}
                    <p id="api-settings-card-description" class="card-description">Your phone number is verified. You can now chat with Khoj on WhatsApp.</p>
                {% elif phone_number and not verified_phone_number %}
                    <p id="api-settings-card-description" class="card-description">Verify your phone number to chat with Khoj on WhatsApp.</p>
```

Reply:
Ah true, good point! The state of the config page should be different for an unverified number.